ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -711,9 +711,6 @@ export const testListPairWithCopyableValue = (): Promise<boolean> =>
           type: WhisperComponentType.ListPair,
           label: 'I am Mr. Label',
           value: copyableText,
-          onValueCopy: () => {
```

Comment:
we will need to create a PR to add these `onValueCopy, onLabelCopy` changes back to develop.

Reply:
Are there any other changes that we're going to need to track,  it sounds like this merge will introduced a break.